db/src/bootstrap.rs

```diff
@@ -195,14 +197,17 @@ fn symbolic_schema_to_triples(ident_map: &IdentMap, symbolic_schema: &Value) ->
                             // bootstrap symbolic schema, or by representing the initial bootstrap
                             // schema directly as Rust data.
                             let typed_value = match TypedValue::from_edn_value(value) {
-                                Some(TypedValue::Keyword(ref s)) => TypedValue::Ref(*ident_map.get(s).ok_or(ErrorKind::UnrecognizedIdent(s.clone()))?),
+                                Some(TypedValue::Keyword(ref s)) => {
+                                    let entid = ident_map.get(&to_namespaced_keyword(s).unwrap());
```

Comment:
If you unwrap here, you should comment about why it's safe… but I think you probably want to not unwrap.